package.json

```diff
@@ -12,37 +12,36 @@
     "test:lighthouse:debug": "node --inspect node_modules/.bin/jest --watch -c ./lighthouse/jest.config.lighthouse.js --runInBand",
     "lint": "npx ng lint clark",
     "e2e": "ng e2e",
-    "storybook": "start-storybook",
-    "prepare": "husky install"
+    "storybook": "start-storybook"
   },
   "@comment dependencies": [
     "DO NOT UPDATE ng2-ckeditor",
     "typescript should be gradually stepped up -- stable as of 2022_09_16"
   ],
   "dependencies": {
-    "@angular/animations": "^13.1.2",
-    "@angular/cdk": "^13.1.2",
-    "@angular/cli": "^13.1.3",
-    "@angular/common": "^13.1.2",
-    "@angular/compiler": "^13.1.2",
-    "@angular/compiler-cli": "^13.1.2",
-    "@angular/core": "^13.1.2",
-    "@angular/forms": "^13.1.2",
-    "@angular/localize": "^13.1.2",
-    "@angular/material": "^13.1.2",
-    "@angular/platform-browser": "^13.1.2",
-    "@angular/platform-browser-dynamic": "^13.1.2",
-    "@angular/router": "^13.1.2",
+    "@angular/animations": "^15.2.2",
+    "@angular/cdk": "^15.2.2",
+    "@angular/cli": "^15.2.2",
+    "@angular/common": "^15.2.2",
+    "@angular/compiler": "^15.2.2",
+    "@angular/compiler-cli": "^15.2.2",
+    "@angular/core": "^15.2.2",
+    "@angular/forms": "^15.2.2",
+    "@angular/localize": "^15.2.2",
+    "@angular/material": "^15.2.2",
+    "@angular/platform-browser": "^15.2.2",
+    "@angular/platform-browser-dynamic": "^15.2.2",
+    "@angular/router": "^15.2.2",
     "@babel/core": "^7.16.7",
     "@cyber4all/clark-taxonomy": "^4.0.5",
     "aws-sdk": "^2.1058.0",
     "core-js": "^3.6.5",
     "md5": "^2.2.1",
     "ng-click-outside": "^9.0.1",
     "ng2-charts": "^2.3.0",
-    "ng2-ckeditor": "1.3.6",
+    "ng2-ckeditor": "1.3.7",
```

Comment:
Was this tested? This can cause breaking changes. Please see the note dependencies note above.

Reply:
+1

Reply:
I downgraded back to v6, hope that helps.